lib/nats.js

```diff
@@ -1131,6 +1132,8 @@ Client.prototype.request = function(subject, opt_msg, opt_options, callback) {
   var inbox = createInbox();
   var s = this.subscribe(inbox, opt_options, function(msg, reply) {
```

Comment:
the `opts_options` can specify the number of responses to receive before auto-unsubscribing. See the jsdoc above.